game/src/graphic/Animation.java

```diff
@@ -74,6 +74,13 @@ public boolean isFinished() {
         return !looping && currentFrameIndex == frames - 1;
     }
 
+    /**
+     * @return true when looping, otherwise false
+     */
+    public boolean isLooping() {
```

Comment:
Versteh nich ganz warum das in diesem PR ist? 
Benutzt du das irgendwo? Ich seh das nicht

Reply:
HealthSystem line 36